qiskit/visualization/circuit/latex.py

```diff
@@ -47,7 +48,24 @@ class QCircuitImage:
     Thanks to Eric Sabo for the initial implementation for Qiskit.
     """
 
-    def __init__(
+    @deprecate_arg("gregs", since="0.20.0")
+    @deprecate_arg("cregs", since="0.20.0")
+    @deprecate_arg("layout", since="0.20.0")
+    @deprecate_arg("global_phase", since="0.20.0")
+    @deprecate_arg(
+        "circuit",
+        deprecation_description=(
+            "Not setting the ``circuit`` argument in the ``QCircuitImage`` constructor"
+        ),
+        since="0.20.0",
+        predicate=lambda circuit: circuit is None,
+        additional_msg=(
+            "The ``circuit`` argument must be a valid QuantumCircuit. If ``circuit`` is set to "
+            "``None``, then the circuit will be built using the ``qubits`` and ``clbits`` "
+            "arguments for rendering the drawing."
+        ),
+    )
+    def __init__(  # pylint: disable=bad-docstring-quotes
```

Comment:
I couldn't figure out why Pylint is mad. Sphinx is happy with the docstring, and I confirmed that `QCircuitImage.__init__.__doc__` looks good.

It seems low priority to figure this out: what we really care about is Sphinx being happy.